modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/object_folder.json

```diff
@@ -0,0 +1,274 @@
+{
+	"actions": {
+		"delete": {
+		},
+		"get": {
+		},
+		"permissions": {
+		},
+		"update": {
+		}
+	},
+	"externalReferenceCode": "ObjectFolder",
+	"label": {
+		"en_US": "ObjectFolder"
+	},
+	"name": "ObjectFolder",
+	"objectFolderItems": [
+		{
+			"linkedObjectDefinition": false,
+			"objectDefinition": {
+				"accountEntryRestricted": false,
+				"accountEntryRestrictedObjectFieldName": "",
+				"actions": {
+					"delete": {
+					},
+					"get": {
+					},
+					"permissions": {
+					},
+					"update": {
+					}
+				},
+				"active": false,
+				"defaultLanguageId": "en_US",
+				"enableCategorization": true,
+				"enableComments": false,
+				"enableLocalization": false,
+				"enableObjectEntryHistory": false,
+				"externalReferenceCode": "customERC",
+				"label": {
+					"en_US": "My Custom"
+				},
+				"modifiable": true,
+				"name": "MyCustom",
+				"objectActions": [
+				],
+				"objectFields": [
+					{
+						"DBType": "String",
```

Comment:
The same of [linked_object_folder.json](https://github.com/liferay-objects/liferay-portal/pull/2939/commits/e9eaa38354d1219b3461684e2f46b617871929d7#diff-04bb5141438dcf2307971b5c4036764323445d3fad3745f97be414dd3bebd0bf)